DataFormats/Detectors/ITSMFT/ITS/include/DataFormatsITS/TrackITS.h

```diff
@@ -122,13 +122,36 @@ class TrackITS : public o2::track::TrackParCov
   void setNextROFbit(bool toggle = true) { setUserField((getUserField() & ~kNextROF) | (-toggle & kNextROF)); }
   bool hasHitInNextROF() const { return getUserField() & kNextROF; }
 
+  void setClusterSize(int l, int size)
+  {
+    if (l >= 8)
+      return;
+    if (size > 15)
+      size = 15;
```

Comment:
strange that this passed the check, could you put brackets?

Reply:
Added